.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,49 @@
+---
+name: "\U+1F56E Bug report"
```

Comment:
should this be updated?

Reply:
Yeah, I had pushed a draft and then noted the lack of update =D You were too fast! I am now using a book symbol.